conference.js

```diff
@@ -2184,21 +2004,6 @@ export default {
             this.hangup(true);
         });
 
-        // logout
-        APP.UI.addListener(UIEvents.LOGOUT, () => {
```

Comment:
I guess we can now remove these from UIEvents.

Reply:
Yeah, those are already gone.